docs/pages/concepts/networking/components/network-manager.md

```diff
@@ -1,21 +1,98 @@
+
+
+### IPAM
+Embedded within the Liqo Network Manager, the IPAM (IP Address Management) is the Liqo module in charge of:
+1. Manage networks and IP addresses, as the name suggests.
+2. Translate IP addresses of offloaded Pods (assigned by foreign clusters) in addresses valid on the home cluster.
+3. Provide IP addresses for endpoints to the Virtual Kubelet during the [Reflection](../../../offloading/features/api-reflection#Overview).
+
+
+#### Networks and addresses management
+Liqo enables Pods in different (peered) clusters to talk one to another.
+Communicating over the Internet requires involved entities to have an IP address; this is not a problem for Pods, as they are assigned an address by the CNI installed on their cluster.
+However, these IPs are valid only within the context of the cluster and therefore are not reacheable by Pods in peered clusters.
+The [Tunnel Operator](../gateway#tunnel-operator) solves this problem, enabling Pods to communicate not only with those running on their same cluster, but also with those running on peered clusters. 
+
+But how does the IPAM come into the rescue?
+Liqo enables traffic toward clusters' PodCIDR but does not make any assumption on those address spaces.
+It is legitimate to imagine a situation in which two peered clusters have the same PodCIDR: in this case two Pods could be assigned the same IP addresses, creating a conflict.
+
+{{% notice info %}}
+The PodCIDR is the network used by the CNI for assigning addresses to Pods.
+The ExternalCIDR is the network used by Liqo for assigning addresses to external resources during Reflection(See more [Here](#reflection-of-external-endpoints)).
+{{% /notice %}}
+
+The Liqo solution is to check if networks (PodCIDR and ExternalCIDR) of a foreign cluster are already in use or are reserved in the home cluster (these networks can be specified in the `clusterconfigs.config.liqo.io` resource) and to choose available networks to map them on, if necessary.
+The solution is implemented by the IPAM module, that keeps track of every network used in the local cluster and remaps remote clusters networks that create conflicts (traffic toward these networks will require NAT operation carried out by the [Tunnel Operator](../gateway#tunnel-operator)).
+
+#### IP addresses translation of offloaded Pods
+Liqo enables to offloading of workloads on peered clusters, giving at the same time the administrator the impression Pods are running on its own cluster.
+The [Virtual Kubelet](../../../offloading#virtual-kubelet) is the component in charge of offloading workloads and keep their status always up to date on the original cluster so that the administrator can inspect it.
+However, offloaded Pods are assigned an IP address by the CNI of the foreign cluster, as they actually are run there.
+Even if these addresses are valid on the foreign cluster, they may have no meaning in the original cluster.
```

Comment:
I would delete this line. Already said before.

Reply:
I would keep the line for 2 reasons:
1. The reader could have skipped the first section, as it was interested only on this part.
2. Since the problem is about translation of IP addresses, it is important when describing the problem to remark who assigns those addresses.